data/data/azure/vnet/master-elb.tf

```diff
@@ -0,0 +1,60 @@
+resource "azurerm_public_ip" "cluster_public_ip" {
+  sku                 = "Standard"
+  location            = "${var.region}"
+  name                = "${var.cluster_id}-pip"
+  resource_group_name = "${var.resource_group_name}"
+  allocation_method   = "Static"
+  domain_name_label   = "${var.dns_label}"
+
+  tags = "${merge(map(
+    "Name", "${var.cluster_id}-pip",
+  ), var.tags)}"
+}
+
+data "azurerm_public_ip" "cluster_public_ip" {
```

Comment:
why [create](https://github.com/openshift/installer/pull/1454/files#diff-493c5bb32fdbabf6b89bf2188dcf314eR1) a resource and then fetch that same resource... ?

Reply:
no goo dreason except I'm iterating and it's not optimal yet.